src/makers/clodpi/hotspots.ts

```diff
@@ -0,0 +1,25 @@
+import HotspotIcon from './clodpi-pro.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const ClodPi = {
+  name: 'ClodPi Pro Hotspot',
+  icon: HotspotIcon,
+  onboardType: 'QR',
+  translations: {
+    en: {
+      diagnostic:
+                '<b><white>Diagnostic support allows ClodPi to identify issues with your Hotspot in a secure way.</white></b>\n\ClodPi will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@clodpi.io</b></purple> from the email used to purchase the Hotspot.',
+      power: [
+        'Attach the antenna and plug in the provided power adapter.',
+        'The ClodPi Pro PWR LED will light up once it’s powered on.',
+      ],
+      externalOnboard:
```

Comment:
Did you want to add an `onboardUrl` so that there is a link to tap?

Reply:
Added onboardUrl

Reply:
Url looks good in the app, although it doesn't load but maybe its not ready yet?